features-json/css-nesting.json

```
@@ -560,7 +560,8 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Available behind the [Experimental Web Platform features](chrome://flags/#enable-experimental-web-platform-features) flag"
+    "1":"Available behind the [Experimental Web Platform features](chrome://flags/#enable-experimental-web-platform-features) flag",
+    "2":"Available behind the `layout.css.nesting.enabled` flag",
```

```suggestion
    "2":"Available behind the `layout.css.nesting.enabled` flag"
```

this should make the tests pass :)